File: python/code/126_pattern18.py
n = int(input("Enter n :- "))
for i in range(n):
    for j in range(i):
        print(" ", end='')
    for k in range(2*(n-i-1)):
        if(k == 0):
            print("*", end='')
        else:
            print(" ", end='')
    print("*")
    
# Output :-
# Enter n :- 5
# *       *
#  *     *
#   *   *
#    * *
#     *

File: c/126_pattern_18.c
#include<stdio.h>
int main() {
    int n;
    printf("Enter n :- ");
    scanf("%d",&n);
    for(int i=0;i<n;i++) {
        for(int j=0;j<i;j++) {
            printf(" ");
        }
        for(int k=0;k<2*(n-i-1);k++) {
            if(k==0) printf("*");
            else printf(" ");
        }
        printf("*\n");
    }
}

/*

Output :-
Enter n :- 5
*       *
 *     *
  *   *
   * *
    *

*/